handbook/admin-tools.md

```diff
@@ -0,0 +1,84 @@
+# Admin Tools in EasyEngine v4
+
+In EasyEngine v4, we have support for following tools to help site administrators:
+
+ * [opcache-gui](#opcache-gui)
+ * [phpinfo](#phpinfo)
+ * [phpMyAdmin](#phpMyAdmin)
+ * [phpRedisAdmin](#phpRedisAdmin)
+ * [MailHog](#MailHog)
+
+## Usage
+In order to use the admin-tools, you must first enable them using 
+```bash
+ee admin-tools enable example.com
+```
+
+By default, we enable auth on admin-tools. To view username/password with which you can login, run
+```bash
+ee auth list global
+```
+
+Then navigate to `example.com/ee-admin/` in browser. That's you'll see a list of admin-tools there. 
+
+## opcache-gui
+[opcache-gui](https://github.com/amnuts/opcache-gui) helps you visualize zend opcache statistics. It showes statistics, settings,cached files, and provide real-time update on information.
+
+## phpinfo
+It is a simple php script with a `phpinfo()` call so you can view information on php like with how many modules it was compiled with and inspect php configuration values. 
+
+## phpMyAdmin
+[phpMyAdmin](https://www.phpmyadmin.net) gives you a GUI to ineract with your database.
+
+On the login screen, you need to add credentials. You can find the credentials by typing 
+```bash
+ee site info example.com
++--------------------+--------------------------------+
+| Site               | https://example.com            |
++--------------------+--------------------------------+
```

Comment:
If you have admin-tools enabled. The link to admin-tools url will be shown here when `ee site info` is invoked.
Enable admin-tools and after that put the output here.